aerospike.tf

```diff
@@ -0,0 +1,188 @@
+  chart                      = try(local.aerospike_cluster_helm_config["chart"], "aerospike-cluster")
+  version                    = try(local.aerospike_cluster_helm_config["version"], var.aerospike_operator_version)
+  timeout                    = try(local.aerospike_cluster_helm_config["timeout"], 300)
+  values                     = try(local.aerospike_cluster_helm_config["values"], null)
+  create_namespace           = try(local.aerospike_cluster_helm_config["create_namespace"], false)
+  namespace                  = try(local.aerospike_cluster_helm_config["namespace"], local.aerospike_namespace)
+  lint                       = try(local.aerospike_cluster_helm_config["lint"], false)
+  description                = try(local.aerospike_cluster_helm_config["description"], "")
+  repository_key_file        = try(local.aerospike_cluster_helm_config["repository_key_file"], "")
+  repository_cert_file       = try(local.aerospike_cluster_helm_config["repository_cert_file"], "")
+  repository_username        = try(local.aerospike_cluster_helm_config["repository_username"], "")
+  repository_password        = try(local.aerospike_cluster_helm_config["repository_password"], "")
+  verify                     = try(local.aerospike_cluster_helm_config["verify"], false)
+  keyring                    = try(local.aerospike_cluster_helm_config["keyring"], "")
+  disable_webhooks           = try(local.aerospike_cluster_helm_config["disable_webhooks"], false)
+  reuse_values               = try(local.aerospike_cluster_helm_config["reuse_values"], false)
+  reset_values               = try(local.aerospike_cluster_helm_config["reset_values"], false)
+  force_update               = try(local.aerospike_cluster_helm_config["force_update"], false)
+  recreate_pods              = try(local.aerospike_cluster_helm_config["recreate_pods"], false)
+  cleanup_on_fail            = try(local.aerospike_cluster_helm_config["cleanup_on_fail"], false)
+  max_history                = try(local.aerospike_cluster_helm_config["max_history"], 0)
+  atomic                     = try(local.aerospike_cluster_helm_config["atomic"], false)
+  skip_crds                  = try(local.aerospike_cluster_helm_config["skip_crds"], false)
+  render_subchart_notes      = try(local.aerospike_cluster_helm_config["render_subchart_notes"], true)
+  disable_openapi_validation = try(local.aerospike_cluster_helm_config["disable_openapi_validation"], false)
+  wait                       = try(local.aerospike_cluster_helm_config["wait"], true)
+  wait_for_jobs              = try(local.aerospike_cluster_helm_config["wait_for_jobs"], false)
+  dependency_update          = try(local.aerospike_cluster_helm_config["dependency_update"], false)
+  replace                    = try(local.aerospike_cluster_helm_config["replace"], false)
+
+  postrender {
+    binary_path = try(local.aerospike_cluster_helm_config["postrender"], "")
+  }
+
```

Comment:
We must add the key

set {
    name  = "watchNamespaces"
    value = "aerospike"
  }
  
  to tell the operator which namespace to be watched.
  
  This is done as per our documentation: 
  https://aerospike.com/docs/cloud/kubernetes/operator/install-operator-helm#2-deploy-the-ako